browser/ui/webui/new_tab_page/brave_new_tab_message_handler.cc

```diff
@@ -10,6 +10,7 @@
 #include <string>
 #include <utility>
 
+#include "base/check_op.h"
```

Comment:
Is this needed?

Reply:
@aseren this was needed before your mojom change, can you please remove. Thanks

Reply:
Fixed